packages/cli/package.json

```diff
@@ -73,7 +73,7 @@
     "@oclif/plugin-not-found": "2.3.23",
     "@oclif/plugin-plugins": "4.1.12",
     "@oclif/plugin-warn-if-update-available": "2.0.24",
-    "@typescript-eslint/typescript-estree": "6.7.2",
+    "@typescript-eslint/typescript-estree": "6.19.0",
```

Comment:
We need to be a bit careful with this one. Each `@typescript-eslint/typescript-estree"` only supports a specific range of TypeScript versions. This can end up being a breaking change for users. (https://github.com/checkly/checkly-cli/issues/848)

I think we can bump it, but if the supported TS versions change it could be good to mention it in the release notes. The supported TypeScript versions are here: https://github.com/typescript-eslint/typescript-eslint/blob/920f909debc3eb394f5aef4e36568bdf6641e304/packages/typescript-estree/src/parseSettings/warnAboutTSVersion.ts#L9.

Reply:
[6.7.2](https://github.com/typescript-eslint/typescript-eslint/blob/v6.7.2/packages/typescript-estree/src/parseSettings/warnAboutTSVersion.ts#L9) only supports up to 5.3 (so 5.3.3 of TS is not supported)  [6.19.0](https://github.com/typescript-eslint/typescript-eslint/blob/v6.19.0/packages/typescript-estree/src/parseSettings/warnAboutTSVersion.ts) supports up to 5.4 and both of them support from 4.3.5 onwards
@tnolet @umutuzgur WDYT?

Reply:
@ferrandiaz this is above my paygrade. Please make the decision based on what's good for the users.

Reply:
Both support 4.3.5 onwards, so we don't risk breaking anyones setup. I think we can just merge it 👍